llama/unicode.cpp

```diff
@@ -227,8 +240,21 @@ static std::unordered_map<std::string, uint8_t> unicode_utf8_to_byte_map() {
 }
 
 static inline std::wstring unicode_wstring_from_utf8(const std::string & s) {
+#ifdef _WIN32
+    int wlen = MultiByteToWideChar(CP_UTF8, 0, s.c_str(), -1, NULL, 0);
```

Comment:
Might be worth checking for `!wlen` before proceeding since one of the cases where that can happen is invalid Unicode.